bin/run_lint.sh

```diff
@@ -17,16 +17,15 @@ PYTHON_VERSION=$(python --version)
 
 
 if [[ $1 == "--fix" ]]; then
-    echo ">>> black fix (${PYTHON_VERSION})"
-    black $FILES
+    echo ">>> ruff fix (${PYTHON_VERSION})"
+    ruff format $FILES
+    ruff check --fix $FILES
```

Comment:
Ah okay so the first command is formatting, while the second command is linting and fixing lint issues.